cli/commands/tessen/opt_in.go

```diff
@@ -0,0 +1,35 @@
+package tessen
+
+import (
+	"errors"
+	"fmt"
+
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/cli"
+	"github.com/spf13/cobra"
+)
+
+// OptInCommand updates the tessen configuration to opt-in
+func OptInCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "opt-in",
+		Short:        "opt-in to the tessen phone home service",
```

Comment:
I _feel_ like "phone home service" makes it sound like some sort of spyware. Maybe we can call it something else? Maybe it's just me though, and I'm not a native English speaker.

Reply:
s/phone home/telemetry/ IMO

Reply:
https://blog.sensu.io/announcing-tessen-the-sensu-call-home-service

call-home vs data collection.. which would you prefer!